src/nvim/message.c

```diff
@@ -3114,14 +3114,14 @@ int vim_vsnprintf(char *str, size_t str_m, char *fmt, va_list ap, typval_T *tvs)
 {
   size_t str_l = 0;
   bool str_avail = str_l < str_m;
-  char        *p = fmt;
+  char *p = fmt;
   int arg_idx = 1;
 
-  if (p == NULL)
+  if (!p)
```

Comment:
Am I the only one who prefers explicitly boolean expressions in control statements (like i.e. Java)? No big deal but our style guideline is not clear in this regard